new french file

cynferdd opened the pull request:
New file added with a list of french cuss words and expressions.
I don't know how to include it in the project so far (didn't find cuss.js in the repository so I don't know how to tell that we may want to use one file or another).

wooorm commented:
@cynferdd Hey this is very nice! Wow!

Could you name the file `fr.json`. After that, we can work on supporting both languages properly :)

`cuss` is based on [`profanities`](https://github.com/words/profanities), so I’d love it if the same list was also there. The tests also check that all (English) profanities are in this project.

Finally, it looks like you didn’t write the list all by yourself. Did you maybe take it from somewhere? If so, we should make sure that it’s actually allowed to use the list, which depends on the license!

wooorm commented:
Oh and also: could you indent the JSON properly, e.g, two spaces. Most text editors let you do that if you select everything and press <kbd>CMD + ]</kbd> (or the like, if you’re not on a mac).

cynferdd commented:
Thanks for the feedback, I'll do those changes asap.

regarding the list, I took it from here : https://fr.wiktionary.org/wiki/Cat%C3%A9gorie:Insultes_en_fran%C3%A7ais 
And I added / modified things on it (and added the value to know if it is a profanity or it may not be). So there should be no issue with it (and those are words commonly used by anyone anyway.)

wooorm commented:
Alright, sweet! Maybe a note about where the words come from in the readme would be good tho 👍
And let me know when you get to it! GitHub doesn’t send a notification if you push new code, so you’ll have to place a comment :)

inesbenomar18 commented on index-fr.json:
```diff
@@ -0,0 +1,737 @@
+"exploser le terrier": 2,
+"face de pet": 2,
+"faciale": 2,
+"faire": 2,
```
As I said in my previous comment, I would give 'faire'  here a 0 rather than a 1 since this is literally just the verb "do" in french

cynferdd replied:
change done. Thanks a lot ! :)

inesbenomar18 commented on index-fr.json:
```diff
@@ -0,0 +1,737 @@
+"se contrefoutre": 2,
+"se désenclaver la péninsule": 2,
+"se doigter": 2,
+"se faire enculer": 2,
```
Here, right after line 623. I would add: 'se faire quelqu'un' or 'se faire' and give it a 2 or 1 since that would be an instance in which the verb do, would translate to a vulgar expression.

cynferdd replied:
change done

wooorm commented:
Thanks for the review @inesbenomar18, that’s awesome!

@cynferdd Let me know if I can help!

cynferdd commented:
Sorry, I have been a bit busy with IRL things, I'll update this either today or tomorrow :)

cynferdd commented:
updates are done, thanks a lot for your feedback. It took me less time than I thought ^^ .

Murderlon commented:
@cynferdd impressive! In terms of logic I think this is ready to be merged, is there anything you'd still like to do?

cynferdd commented:
nothing to add on this project so far. The next thing I'll update will be retext-profanities (which uses cuss).

wooorm commented:
Awesome, looks good! 👍

wooorm commented:
@cynferdd Oh one thing I forgot: could you describe where this list comes from in the readme? (see https://github.com/words/cuss/pull/15#issuecomment-426917761)

cynferdd commented:
oh you're right ! I totally forgot about that, sorry. I add it right now.

cynferdd commented:
it's updated. Thanks for the information about npm test.

wooorm commented:
Thanks @cynferdd for this awesome list!! 🎉
